graphql2/graphqlapp/alert.go

```diff
@@ -361,6 +360,28 @@ func (m *Mutation) CreateAlert(ctx context.Context, input graphql2.CreateAlertIn
 	return m.AlertStore.Create(ctx, a)
 }
 
+func (a *Alert) NoiseReason(ctx context.Context, raw *alert.Alert) (*string, error) {
+	am, err := a.AlertStore.Feedback(ctx, raw.ID)
+	if err != nil {
+		return nil, err
+	}
+	return &am.NoiseReason, nil
```

Comment:
According to the schema, this is a null-able field, so we should probably return `nil` here if `NoiseReason == ""`